src/elements/design-principles.md

```diff
@@ -0,0 +1,55 @@
+# Design Principles
+
+The following conventions describe the Python implementation. Matlab conventions are similar and will be described separately.
```

Comment:
Where are we describing Matlab conventions? Maybe we should just link to our matlab docs here?
```suggestion
The following conventions describe the Python implementation. 
```

Reply:
We will need to describe the conventions for `element-visual-stimulus`.  We can remove this note for now.